packages/components/List/Item/styles.js

```diff
@@ -133,7 +136,7 @@ export const Row = styled.div`
   }
   
   ${IconWrapper} {
-    display: ${({ isMultiItem }) => (isMultiItem ? 'none' : 'inline-block')};
+    display: ${({ isMultiItem }) => (isMultiItem ? 'none' : 'flex')};
```

Comment:
maybe, inline-flex?
// i'm not sure, but seems that inline-block was here for a reason
// nitpick

Reply:
IconWrapper is a span element and `inline-block` was used in opposite of `none` for display property. So now `flex` works fine here